src/main/java/org/spongepowered/api/entity/Entity.java

```diff
@@ -122,12 +140,74 @@
      * <ul><code>x -> pitch</code>, <code>y -> yaw</code>, <code>z -> roll
      * </code></ul>
      *
-     * @param rotation The rotation to set the entity to
+     * @param rotation The rotation to set
      */
     void setRotation(Vector3d rotation);
 
     /**
-     * Moves the entity to the specified location, and sets the rotation.
+     * Moves this entity to the specified position in its current world, and sets the rotation.
+     *
+     * <p>The format of the rotation is represented by:</p>
+     *
+     * <ul><code>x -> pitch</code>, <code>y -> yaw</code>, <code>z -> roll
+     * </code></ul>
+     *
+     * @param position The position to set
+     * @param rotation The rotation to set
+     */
+    void setPositionAndRotation(Vector3d position, Vector3d rotation);
```

Comment:
I wonder if instead of making these combined methods we could just make the setters fluent.